src/pages/top-speaker/list/index.tsx

```diff
@@ -124,12 +132,16 @@ const TopSpeakerList: React.FC = () => {
 						<span className="dropdown-item" onClick={() => handleSetViewTopSpeaker(row._id)}>
 							<i className="far fa-eye" /> View
 						</span>
-						<span className="dropdown-item" onClick={() => handleSetUpdateTopSpeaker(row._id)}>
-							<i className="far fa-edit" /> Edit
-						</span>
-						<button className="dropdown-item" onClick={() => handleSetDeleteTopSpeaker(row._id)}>
-							<i className="far fa-trash-alt" /> Delete
-						</button>
+						{(permission === "ROOT_ADMIN" || permission === "ADMIN") && (
```

Comment:
add `permission === "EDITOR"` as well. Because the editor should be able to edit the top speaker information.